release_notes/371-release-notes.adoc

```diff
@@ -0,0 +1,225 @@
+
+In the table, features are marked with the following statuses:
+
+- GA: General Availability
+- TP: Technology Preview
+- DEP: Deprecated
+- REM: Removed
+
+.Deprecated and removed features tracker
+[cols="3,1,1,1",options="header"]
+|====
+|Feature |{product-title-short} 3.69 |{product-title-short} 3.70 | {product-title-short} 3.71
+
+|Ability to add comments to alerts and processes
+|DEP
+|DEP
+|REM
+
+|Anchore, Tenable, and Docker Trusted registry integrations
+|DEP
+|DEP
+|REM
+
+|External authorization plug-in for scoped access control
+|DEP
+|DEP
+|REM
+
+|`FROM` option in the Disallowed Dockerfile line policy field
+|GA
+|DEP
+|DEP
+
+| `PodSecurityPolicy` (PSP) Kubernetes objects
+| GA
+| DEP
+| REM
+
+|`RenamePolicyCategory` and `DeletePolicyCategory` API endpoints
+|GA
+|DEP
+|DEP
+
+| `--rhacs` option for the `roxctl` helm output command
+| DEP
+| DEP
+| REM
+
+|`/v1/nodes` and `/v1/images` API endpoint response: `firstNodeOccurrence` response body parameter
+|DEP
+|DEP
+|REM
+
+
+| Permissions: `AuthPlugin`, `AuthProvider`, `Group`, `Licenses`, `Role`, `User`, `Indicator`, `NetworkBaseline`, `ProcessWhitelist`, `Risk`, `APIToken`, `BackupPlugins`, `ImageIntegration`, `Notifier`, `SignatureIntegration`, `ImageComponent`
+| GA
+| GA
+| DEP
+
+| Retrieving groups by property
+| GA
+| GA
+| DEP
+
+|====
+
+[id="deprecated-features-371"]
+=== Deprecated features
+
+* Permissions for permission sets will be grouped for simplification. The following list describes the new permissions and indicates the deprecated permissions that will be removed in a future release:
+** New permission Access will replace the permissions `AuthPlugin`, `AuthProvider`, `Group`, `Licenses`, `Role`, and `User`.
+** New permission `DeploymentExtension` will replace the permissions `Indicator`, `NetworkBaseline`, `ProcessWhitelist`, and `Risk`.
+** New permission Integration will deprecate the permissions `APIToken`, `BackupPlugins`, `ImageIntegration`, `Notifier`, and `SignatureIntegration`.
+** New permission Image will replace the permission `ImageComponent`.
+* Previously, groups were retrieved by the field props (`props.authProviderId`, `props.key`, `props.value`). This field will be replaced by the new `props.id` field. Use the props.id field to retrieve groups in the Application Programming Interface (API). Note the following:
+** Retrieval via the `props` fields will be removed in version {product-title-short} version 3.73.
+** Until removal, retrieval via the `props` field will work as long as the result is unambiguous (no more than one group is found with the  props field).
```

Comment:
Regarding "as long as": Do not use except to refer to a comparison of distance, length, or time, as in "set the validity period to be as long as you want". Otherwise, use "while" to refer to time, or use "if" or "when" to mean "on condition that".

I think we can use "if" here.